hack/tool/tackle

```diff
@@ -299,19 +188,118 @@ class TackleTool:
         print("ERROR: %s record ID %d not found." % (objType, id))
         exit(1)
 
+    def checkTokenLifetime(self):
+        if self.TokenRenewAfter < int(time.time()):
+             self.Token =  getHubToken(self.Url, c['username'], c['password'], False)
+             self.TokenRenewAfter = int(time.time()) + 4*60 # 4 minutes to refresh token
```

Comment:
Shouldn't the token have this information? If an auth provider has a shorter refresh token, this will fail, I think.

Reply:
You're right. The supported go-based implementation also uses Refresh call instead of new Login (https://github.com/konveyor/tackle2-hub/blob/main/binding/client.go#L781).

For this semi/un-supported Python script, that is planned to be replaced with a golang-based tooling, this could be a quickfix IMO.

The time was moved to constant to allow easier adjustment in case of problems.

Reply:
quick fix makes sense, was just a concern, if this is not going to live for much longer I am good :)